digdag-tests/build.gradle

```diff
@@ -4,8 +4,8 @@ dependencies {
     testCompile project(':digdag-storage-s3')
     testCompile 'com.google.code.findbugs:annotations:3.0.1'
     testCompile 'org.subethamail:subethasmtp:3.1.7'
-    testCompile 'com.squareup.okhttp3:okhttp:3.4.1'
-    testCompile 'com.squareup.okhttp3:mockwebserver:3.4.1'
+    testCompile 'com.squareup.okhttp3:okhttp:3.9.0'
+    testCompile 'com.squareup.okhttp3:mockwebserver:3.9.0'
```

Comment:
[td-client-java uses okhttp 3.9](https://github.com/treasure-data/td-client-java/blob/4e45321e89a87e6d975db496871d994dfbb51544/pom.xml#L97-L112), so I also updated okhttp used in digdag.